easybuild/scripts/rpath_args.py

```diff
@@ -119,6 +118,14 @@
 
     idx += 1
 
+# also inject -rpath options for all entries in $LIBRARY_PATH,
+# unless they are there already
+for lib_path in os.getenv('LIBRARY_PATH', '').split(os.pathsep):
+    if lib_path and (rpath_filter is None or not rpath_filter.match(lib_path)):
```

Comment:
How come you don't do the absolute path checking here?

Reply:
Well, I could I guess, but I'm assuming only absolute paths are listed in `$LIBRARY_PATH`, which seems like a fair assumption?

Reply:
Actually, consistent is better, so fixed in c35647f